packages/integration-tests/lambda.js

```diff
@@ -2,10 +2,26 @@
 
 const { lambda } = require('@cumulus/aws-client/services');
 
+/**
+ * Retrieve a rule's Kinesis Event Source Mappings
+ *
+ * @param {Object} uuid - unique identifier for a rule
```

Comment:
Should this be a string?

Reply:
Yup. This was re-written and not updated:
https://github.com/nasa/cumulus/pull/3149/commits/4df26b664bfa0933ca41ddc604bf6a12222af1de